eclass/mozcoreconf-v6.eclass

```diff
@@ -76,18 +76,6 @@ mozconfig_use_with() {
 	mozconfig_annotate "$(use $1 && echo +$1 || echo -$1)" "${flag}"
 }
 
-# @FUNCTION: mozconfig_use_extension
-# @DESCRIPTION:
-# enable or disable an extension based on a USE-flag
-#
-# Example:
-# mozconfig_use_extension gnome gnomevfs
-# => ac_add_options --enable-extensions=gnomevfs
-mozconfig_use_extension() {
-	declare minus=$(use $1 || echo -)
-	mozconfig_annotate "${minus:-+}$1" --enable-extensions=${minus}${2}
-}
-
```

Comment:
Do we *need* to remove this option now?

Reply:
Not sure about this part, but the part below passes the option flag, even if empty, to the seamonkey configure script. This causes the seamonkey configure script to fail due to the flag being present. Do note that this configure option is obsolete for quite some time (has only been in there to support the flash plugin, which Gentoo doesn't have any more).

Reply:
Yeah I was looking into just disabling the option, but I guess this is fine too.